package.json

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
I get confused here because it looks like you are replacing the old framework (using fs.go live in dev mode and pregenerating files in prod mode) but also started using normal react workflow (I assume run node to generate babel in dev mode, and yarn to pregenerate in prod mode). How does that work?

Reply:
So at the moment I'm really only using normal react workflow to run the tests. Once we have a decent bit of test coverage it will make for a pretty easy transition over to the react workflow without having to mess with the existing framework in the mean time. So in the meantime we could have this separate node/yarn set up for running the tests and handling dependencies.

Reply:
Ah, I see - I thought you started using node as dev server and live reload and everything :D